.env.dev

```diff
@@ -112,3 +112,9 @@ FLASK_ENV=development
 # Flask secret_key, see http://flask.pocoo.org/docs/1.0/quickstart/#sessions
 # python -c 'import os; import binascii; print(binascii.hexlify(os.urandom(32)))'
 FLASK_SECRET_KEY=@E\xab\xf5\xddR\xa7\xfbO\xc7\xe3\x9d\x1e\xb4\x06\xe6\x18\x9eP\x18V(\tQ
+
+##############################################################################
+# QGIS settings #############################################################
```

Comment:
There already is QGIS section called "# connection parameters to QGIS Server" (in dev and test env files, not yet in demo). Those two sections should be merged into one.